examples/farmer/farmer_rho_demo.py

```diff
@@ -72,14 +76,16 @@ def main():
 
     num_scen = cfg.num_scens
     crops_multiplier = cfg.crops_mult
-    rho_setter = None
+    rho_setter = None  # non-grad rho setter?
```

Comment:
I think you can drop the "?" - I believe you are correct.

Reply:
I agree